homer0/projext
# Update dependencies

### What does this PR do?

It updates all the project production and development dependencies to their latest version.

### How should it be tested manually?

This change is not breaking as the only dependency being updated to a new major version is `fs-extra` and the change is for `copy *`, a method projext doesn't use.

Run projext as usual and everything should be work... and of course:

```bash
yarn test
# or
npm test
```
